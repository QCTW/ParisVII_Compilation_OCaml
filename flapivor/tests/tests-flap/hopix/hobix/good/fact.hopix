fun fact (n) = if n = 0 then 1 else fact (n - 1) * n

val x = fact (6)
