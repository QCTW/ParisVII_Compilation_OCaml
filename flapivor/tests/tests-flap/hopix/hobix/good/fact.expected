fact = <fun>
x = 720
